scripts/commands/updateApiDocs.ts

```diff
@@ -383,15 +380,15 @@ async function downloadApiSources(
   destination: string,
   listenPort: number,
 ) {
+  await startWebServer(`${destination}/artifact`, listenPort);
   try {
     await downloadCIArtifact(pkg.name, artifactUrl, destination);
     await saveHtml({
       baseUrl: pkg.baseUrl,
       initialUrl: pkg.initialUrl,
       destination,
     });
-  } catch (e) {
+  } finally {
     await closeWebServer(listenPort);
-    throw e;
```

Comment:
Nice! I was going to make an issue for this; I had the server linger a few times when I encountered an error between this function and the end of `zxMain`.